### PR TITLE
Ignore Ingresses that have spec.ingressClassName

### PR DESCRIPTION
**Pull request by swetharepakula**
- if annotation is present, controller will manage the ingress

cc/ @spencerhance

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ func IsGCEIngress(ing *v1beta1.Ingress) bool {
 
 	switch class {
 	case "":
-		return true
+		return ing.Spec.IngressClassName == nil
```
Comment (aledbf) [inline pkg/utils/utils.go:288]: How we can start using IngressClassName instead of annotations?

**Review by spencerhance — APPROVED** [review]
lgtm.  We may want to add support for gce IngressClass sooner, but can be done separately.

File: pkg/utils/utils.go (hunk 1 above)
Comment (swetharepakula) [reply pkg/utils/utils.go:288]: We will be supporting ingressClassName. #1301 exists to track. This PR is just to make sure not to operate on Ingresses that are meant for other controllers like Istio.

Comment (freehan) [inline pkg/utils/utils.go:288]: Nit: add a comment here to explain the reasoning

**Comment by freehan** [comment]
LGTM overall

**Review by freehan — APPROVED** [review]
/lgtm
/approve

**Pull request merged**